corehq/apps/reports/filters/base.py

```diff
@@ -39,6 +41,8 @@ def __init__(self, request, domain=None, timezone=pytz.utc, parent_report=None,
         self.css_label = css_label or (CSS_LABEL_CLASS + ' control-label')
         self.css_field = css_field or CSS_FIELD_CLASS
         self.context = {}
+        if use_bootstrap5:
+            self.template = get_bootstrap5_path(self.template)
```

Comment:
I think this currently works, but it seems more durable if this code was moved before the check to ensure that `self.template` isn't `None`. I assume if `get_bootstrap5_path` somehow took a non-None template and made it `None`, that should raise an error here.

Reply:
`get_bootstrap5_path` cannot take a string and make it `None` it only returns `None` if it's passed `None`. Otherwise it just replaces `bootstrap3` with `bootstrap5`, if present.